src/components/MySelect.tsx

```diff
@@ -0,0 +1,28 @@
+
+import { SelectChangeEvent, FormControl, InputLabel, MenuItem, Select } from "@mui/material";
+import React from "react";
+
+type SelectPropsType = {
```

Comment:
```suggestion
type Props = {
```
これは慣例ですが、一つのファイルに存在するreact componentが一つのとき、propsの型エイリアスの名前はただのPropsで十分です。
今回はMySelectコンポーネントだけですが、これと関連性が強いコンポーネントをこのファイルに混在させた場合だと、propsの型を区別できるように工夫する必要がありますね！